client/stylesheets/_variables.scss

```diff
@@ -9,3 +9,21 @@ $gap-smallest: 4px;
 
 // @todo remove this spacing variable
 $spacing: 16px;
+
+// Gutenberg Button variables. So very temporary
+$default-font-size: 16px;
+$blue-medium-200: #bfe7f3;
+$light-gray-500: #e2e4e7;
+$dark-gray-300: #6c7781;
+$dark-gray-900: #191e23;
+$alert-red: #d94f4f;
+
+@mixin button-style__focus-active() {
```

Comment:
Is this mixin temporary as well? If not, it should go in the mixins file instead.